packages/termux-keyring/build.sh

```diff
@@ -33,3 +34,13 @@ termux_step_make_install() {
 		echo "998DE27318E867EA976BA877389CEED64573DFCA:4:" > $GPG_DIR/termux-pacman-trusted
 	fi
 }
+
+termux_step_create_debscripts() {
+	if [ "$TERMUX_PACKAGE_FORMAT" = "pacman" ]; then
+		echo "if [ ! -d $TERMUX_PREFIX/etc/pacman.d/ ]; then" > postupg
+		echo "  pacman-key --init" >> postupg
+		echo "fi" >> postupg
+		echo "pacman-key --populate" >> postupg
+		echo "post_upgrade" > postinst
```

Comment:
`postinst` here should probably be `postupg`?

Reply:
Or maybe not considering it uses > instead of >>, I am not familiar enough with how pacman works

Reply:
Looking at your other PR I guess last line here shouldn't be here at all?

Reply:
No, since `postupg` only works when the package is updated. The `termux-keyring` package needs this action to be run after installation as well.